suite-native/device-manager/src/components/DeviceItem/DeviceItem.tsx

```diff
@@ -0,0 +1,37 @@
+import { useSelector } from 'react-redux';
+import { Pressable } from 'react-native';
+
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { HStack } from '@suite-native/atoms';
+import { DeviceRootState, selectDeviceById } from '@suite-common/wallet-core';
+import { Icon } from '@suite-common/icons';
+import { TrezorDevice } from '@suite-common/suite-types';
+
+import { DeviceItemContent } from './DeviceItemContent';
+
+type DeviceItemProps = {
+    id: TrezorDevice['id'];
+    onPress?: () => void;
+};
+
+const deviceItemWrapperStyle = prepareNativeStyle(utils => ({
+    justifyContent: 'space-between',
+    alignItems: 'center',
+    paddingVertical: 10,
+    paddingHorizontal: utils.spacings.medium,
+}));
+
+export const DeviceItem = ({ id: deviceItemId, onPress }: DeviceItemProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const device = useSelector((state: DeviceRootState) => selectDeviceById(state, deviceItemId));
+
+    return (
+        <Pressable onPress={onPress}>
+            <HStack style={applyStyle(deviceItemWrapperStyle)}>
+                <DeviceItemContent deviceState={device?.state} />
```

Comment:
Remove selector for device as we don't need it here, send device state to this component and use that as prop here please. Selecting device by id seems redundant...

Reply:
good spot